commons/src/lib.rs

```diff
@@ -25,9 +25,21 @@ pub mod prelude_errors {
 }
 
 use actix_web::http::{header, HeaderMap};
+use std::collections::HashMap;
 use std::collections::HashSet;
 use url::form_urlencoded;
 
+lazy_static! {
+    /// list of cincinnati versions
+    pub static ref CINCI_VERSION: HashMap<&'static str, i32> =
+        [("application/vnd.redhat.cincinnati.v1+json", 1)]
+            .iter()
+            .cloned()
+            .collect();
+    /// minimum cincinnati version supported
+    pub static ref MIN_CINCI_VERSION: i32 = 1;
```

Comment:
nit: are we already scoped to Cincinnati by our package?  I am not clear on why we need `CINCI` here.  And while I like the Cincy abbreviation internally, I'd rather spell out `Cincinnati` in formal public APIs instead of abbreviating, if it turns out that package scoping isn't enough to allow `MIN_VERSION` here.